### ISSUE TITLE
Certain areas uses old profiling icons (about:profiling & Process Manager)

### ISSUE THREAD

**Issue by Aleksandra0056** [black7375/Firefox-UI-Fix#918]
### Describe the bug

I noticed this much earlier, but now, a few months after Lepton icons re-design, I finally decided to report this issue here.

The about:profiling page uses a non-Lepton icon for the header image (because the mentioned icon belongs to older Photon icons).

I reported a similar issue on Bugzilla ([1738529](https://bugzilla.mozilla.org/show_bug.cgi?id=1738529)), but that doesn't apply to this since clean Firefox doesn't use Lepton icons.

More affected areas are explained in the comments below.

----------

Appropriate Lepton icon for replacement (performance.svg):

![performance](https://github.com/black7375/Firefox-UI-Fix/assets/69351811/151712d9-0ba4-4953-8e1c-b17a8af46398)

NOTE: That icon appears vertically mirrored in some areas for some reason.

### Expected behavior

The about:profiling page should use the same icon (which is on Developer Ed. & Nightly toolbar) as an icon of a header image.

#### Screenshot of the image in about:profiling header:
![The aboutProfiling page uses non-Lepton icon for header image](https://github.com/black7375/Firefox-UI-Fix/assets/69351811/21013492-fb20-412c-9af9-23fbf182df27)

### OS

Windows 10

### Firefox Version

Nightly (64-bit)

### Distribution

- [X] [Original Lepton](https://github.com/black7375/Firefox-UI-Fix)
- [ ] [Lepton's photon style](https://github.com/black7375/Firefox-UI-Fix/tree/photon-style)
- [ ] [Lepton's proton style](https://github.com/black7375/Firefox-UI-Fix/tree/proton-style)

### Theme

- [X] Light
- [X] Dark
- [ ] Alpenglow (System: Light)
- [ ] Alpenglow (System: Dark)
- [ ] System Default
- [ ] Others: (e.g. [Photon Colors](https://addons.mozilla.org/en-US/firefox/addon/photon-colors/), [Google Chrome Dark](https://addons.mozilla.org/en-US/firefox/addon/google-chrome-dark/), [Fairytale Of Nature](https://addons.mozilla.org/en-US/firefox/addon/fairytale-of-nature/))

### `user.js` setup

<details><summary><strong><code>user.js</code> setup</strong></summary><br>

```javascript
// I can't upload the user.js setup data since my Nightly instance was on Windows Sandbox VM.
```
</details>

**Comment by Aleksandra0056** [comment]
This also applies to the tab title icon:
![This also applies to the tab title icon](https://github.com/black7375/Firefox-UI-Fix/assets/69351811/0967ebb2-bb0b-47f3-9e94-601d614ef876)

**Comment by Aleksandra0056** [comment]
Two old profiling icons are also located in Process Manager:

One is on the Process Manager tab title:
![Process Manager tab title icon](https://github.com/black7375/Firefox-UI-Fix/assets/69351811/499a1dbb-e970-4199-80c4-33c98bb0ae07)

The other is on "Profile all threads of this process for 5 seconds" item:
![The profile all threads of this process for 5 seconds uses old icon](https://github.com/black7375/Firefox-UI-Fix/assets/69351811/3e4a8eb1-46f5-44c3-a33c-e9a64a2f0f36)

**Comment by Aleksandra0056** [comment]
Thank you, it's now fixed.
